app/tests/spec/views/mixins/settings-panel-mixin.js

```diff
@@ -76,6 +76,30 @@ define(function (require, exports, module) {
         assert.isFalse(view.isPanelOpen());
       });
 
+      it('openPanel focuses the first input field or button if present', function () {
+        view.openPanel();
+        var input = $('.open input:first-of-type');
+        if (input.length > 0) {
```

Comment:
This test is a bit wacky. One of these branches is going to run, and the other is not. It seems like this should be two tests, one that  only contains the first branch, one that only contains the second branch.